webpack.config.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * TODO: Build this out.
+ * 
+ * First setup webpack with TypeScript:
+ * https://webpack.js.org/guides/typescript/
+ * 
+ * Then add Sass:
+ * https://webpack.js.org/loaders/sass-loader/
```

Comment:
Once you have TS setup you can add Sass, which is by-and-large the most popular CSS flavor.